web/components/TextBlock/SharedSections/SharedSections.tsx

```diff
@@ -20,34 +23,53 @@ export const SharedSections = ({ value: { name, sections, ...rest } }) => (
   <>
     <Heading type="h2">{name}</Heading>
     {sections.map((section) => {
+      // Matches /studio/schemas/sections/index.ts
+      // plus "figure" from /studio/schemas/documents/sharedSections.ts
       switch (section._type) {
-        case 'mailchimp':
-        case 'mailchimpSection':
+        case 'figure':
+          return <Figure key={section._key} value={section} />;
```

Comment:
Any reason this is still using an inlined `Figure`? Can it use the proper Figure component instead (currently in web/components/TextBlock/Figure/`?

Reply:
Great catch. 3077d0d.